APISpecification.yaml

```diff
@@ -111,6 +111,55 @@ paths:
         404:
           description: Genome not found
           content: {}
+  /api/metadata/genome/{genome_id}/ftplinks:
+    get:
+      summary: Returns a list of FTP download links for the specified genome
+      parameters:
+      - name: genome_id
+        in: path
+        required: true
+        schema:
+          type: string
+          example: 3704ceb1-948d-11ec-a39d-005056b38ce3
+      responses:
+        200:
+          description: Successful operation
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/FTPLink'
+        404:
+          description: Genome not found
+          content: {}
+  /api/metadata/genome/{genome_id}/checksum/{region_id}:
```

Comment:
Do we need schema for this API as well?

Reply:
For top-level checksums? There's a [ticket ](https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-2423) for it.

Reply:
Yes. What I mean was that, do we need to add something like line 825?

Reply:
Ah, do you mean using the `FTPLink` schema component instead of including the spec inline? I think moving 18 lines of spec to its own component keeps the main endpoint spec cleaner, and also is consistent with rest of the document (e.g. see `example_objects` endpoint which uses even smaller component).